packages/bits/src/constants/keycode.constants.ts

```diff
@@ -23,6 +23,8 @@ export enum KEY_CODE {
 
 export enum KEYBOARD_CODE {
```

Comment:
There is a collection of key constants in the cdk library here: https://github.com/angular/components/blob/master/src/cdk/keycodes/keycodes.ts
I'd prefer to use that instead of a custom enum.

Reply:
keyCode was deprecated few years ago. That why I prefer to avoid using it.
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode